app/models/exercise_status.rb

```diff
@@ -1,4 +1,6 @@
 class ExerciseStatus < ActiveRecord::Base
```

Comment:
ExerciseStatus is not the appropriate model name if we're following rails conventions. This is a intervening lookup table between users and exercises, hence it should be UserExercises. The attribute on UserExercises would be status. Status field on ExerciseStatus sounds weird.